### PR TITLE
Change `protection` level of `ApplicationEngine.LoadContext`

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/3239

### CODE REVIEW

**Review by Jim8y — APPROVED** [review]

File: src/Neo.VM/ExecutionEngine.cs [hunk 1]
```diff
@@ -166,7 +166,7 @@ protected internal void ExecuteNext()
         /// Loads the specified context into the invocation stack.
         /// </summary>
         /// <param name="context">The context to load.</param>
-        internal virtual void LoadContext(ExecutionContext context)
+        protected internal virtual void LoadContext(ExecutionContext context)
```
Comment (cschuchardt88) [inline src/Neo.VM/ExecutionEngine.cs:169]: If they both need to be `internal` then make it `public`. When a `3rd-party` does `public class MyApplicationEngine : ApplicationEngine` it needs to work with `inheritances`. For example `neo-express` uses this and I can't `override` the method so I can update for latest release of `neo`.

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]

**Comment by cschuchardt88** [comment]
@shargon what happen to `12` hours waiting time. This PR isn't going to work.

File: src/Neo.VM/ExecutionEngine.cs (hunk 1 above)
Comment (shargon) [reply src/Neo.VM/ExecutionEngine.cs:169]: it can be extended by neo-express with protected internal

**Comment by shargon** [comment]
You are right, it should be public, @cschuchardt88 please open a new pr

**Comment by cschuchardt88** [comment]
> You are right, it should be public, @cschuchardt88 please open a new pr

@shargon see #3243

**Pull request merged**